workers/ohsome_quality_analyst/geodatabase/create_results_table.sql

```diff
@@ -0,0 +1,13 @@
+CREATE TABLE IF NOT EXISTS results (
+    indicator text,
```

Comment:
Be careful with the word `INDICATOR` as this seems to be a reserved word in SQL (although I'm not sure about PostgreSQL). If you need more inside about why and how PostgreSQL deals with SQL-reserved-words: https://www.postgresql.org/docs/current/sql-keywords-appendix.html

P.S.: the same applies to `VALUE`

Reply:
Thats a good point.
Two solutions come to mind:
- Quote those identifiers: `"indicator"`
- Rename them to: `indicator_` or `_indicator`
- Rename them to: `indicator_name` and `val`

Do you have a preference here?

Reply:
as the value is representation the indicator name, I would use `indicator_name`.

Reply:
I renamed attributes to avoid usage of SQL keywords